setup_wizard.sh

```diff
@@ -66,18 +66,29 @@ if [ -z "$STORAGE_SIZE" ]; then
   exit 1
 fi
 
+echocol "...."
+echocol "Selecting a SHM for Postgres..."
+POSTGRES_SHM=$(./set_postgres_shm.sh echo-value)
+
+if [ -z "$POSTGRES_SHM" ]; then
+  echo "Error encountered when estimating postgres container shm, exiting"
+  exit 1
+fi
+
 echocol "...."
 echocol "The following parameters will be saved to your .env file. Press ENTER to confirm or quit with CONTROL-C to abort:"
 echo "RLN_RELAY_ETH_CLIENT_ADDRESS='$RLN_RELAY_ETH_CLIENT_ADDRESS'
 ETH_TESTNET_KEY=$ETH_TESTNET_KEY
 RLN_RELAY_CRED_PASSWORD='$RLN_RELAY_CRED_PASSWORD'
-STORAGE_SIZE=$STORAGE_SIZE"
+STORAGE_SIZE=$STORAGE_SIZE
+POSTGRES_SHM=$POSTGRES_SHM"
```

Comment:
Nitpick comment aside from the PR's purpose:
In general, I think it is better to refer variables with curly braces
```suggestion
POSTGRES_SHM=${POSTGRES_SHM}"
```